src/jenkins_job_insight/notifications.py

```
@@ -0,0 +1,106 @@
+                    and exc.response is not None
+                    and exc.response.status_code == 410
+                ):
+                    logger.info(
+                        f"send_mention_notifications: stale subscription (410) for {sub['username']}, scheduling removal"
+                    )
+                    stale_endpoints.append(sub["endpoint"])
+                else:
+                    logger.warning(
+                        f"send_mention_notifications: failed to send to {sub['username']}: {exc}"
+                    )
+            except Exception as exc:
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `Exception`

(BLE001)